charts/kubewarden-controller/Chart.yaml

```diff
@@ -40,7 +40,7 @@ annotations:
   catalog.cattle.io/requests-cpu: "250m"
```

Comment:
(opening thread here because GH doesn't allow on line 36)

Being pedantic, the `kubewarden-crds` chart `version` and `appVersion` should have been bumped, and then both `kubewarden-controller` and `kubewarden-defaults` should have `catalog.cattle.io/auto-install: kubewarden-crds=1.4.0-rc2`.

I will add a commit with that.

Reply:
I think you're right. Let's fix that manually on this PR and track that with a dedicated issue

Reply:
Fixed with followup commits.